addon/components/osf-paginator/component.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+import layout from './template';
+
+// Ember component that is very similar to osf/website/static/js/paginator.js
```

Comment:
Even though I don't want us to get too attached to the old OSF pagination code, we should document the intended API (including parameters required to use it), to support future refactoring.

Otherwise, there are variables used (but not defined) in this code- it should be clear where data comes from.

Most of our other components have docstrings that can be used as examples. Can you please add something similar here following the pattern set out elsewhere in the ember-osf codebase?